### ISSUE TITLE
if I want to train this model

### ISSUE THREAD

**Issue by iperov** [shaoanlu/face-segmentation-keras#2]
what type of optimizer and loss func I should to use ?
